src/app/components/list/list.js

```diff
@@ -122,8 +121,6 @@ listControllers.controller('ListCtrl', ['$scope', '$routeParams', '$location', '
       });
     };
     $scope.list.$promise.then(listCallback);
-    $scope.list.$httpPromise.then(listCallback);
```

Comment:
It didn't seem necessary (I may well be wrong) and was making the users be requested from the api twice.